assignments/this.js

```diff
@@ -13,14 +13,52 @@
 
 // code example for Window Binding
 
+function example1(){
```

Comment:
You cold console log this outside of the function and return the global binding of this